app/views/articles/show.html.erb

```diff
@@ -114,7 +114,10 @@
                 <div class="pl-3 flex-1">
                   <a href="/<%= @user.username %>" class="crayons-link fw-bold"><%= @user.name %></a>
                   <% if @organization %>
-                    <span class="color-base-60">for</span> <a href="<%= @organization.path %>" class="crayons-link"><%= @organization.name %></a>
+                    <%= t("views.articles.for_org_html",
+                          start: tag("span", { class: "color-base-60" }, true),
```

Comment:
Reading notes the third parameter (e.g., `true`) means to only open the tag and not close it.  Hence the `:start` and `:end` key